### PR TITLE
Unit test cases have been updated for SDL-CT196 requirements.

### PR DESCRIPTION
**Pull request by vsivar1x**
Unit test cases have been updated for SDL-CT196 requirements.
Signed-off-by: Sivaramakrishnan, VenkataramananX <venkataramananx.sivaramakrishnan@intel.com>

### CODE REVIEW

File: tests/opae-c/test_buffer_c.cpp [hunk 1]
```diff
@@ -122,4 +122,43 @@ TEST_P(buffer_c_p, ioaddr) {
   EXPECT_EQ(fpgaReleaseBuffer(accel_, wsid), FPGA_OK);
 }
 
+/**
+ * @test       fpgaGetIOAddress
+ * @brief      Test: fpgaGetIOAddress_neg_test
+ * @details    When called with a invalid fpga handle,<br>
+ *             the method returns <br>
```
Comment (tswhison) [inline tests/opae-c/test_buffer_c.cpp:129]: returns ?

**Review by tswhison — COMMENTED** [review]
If the goal of this was to increase coverage, then we have not achieved it.  The Coveralls report says "Overall coverage decreased (-0.04%) to 68.367%"

File: tests/opae-c/test_buffer_c.cpp (hunk 1 above)
Comment (vsivar1x) [reply tests/opae-c/test_buffer_c.cpp]: Updated.

**Comment by r-rojo** [comment]
I would guess that if the coverage doesn't increase then these negative tests may already be covered by other tests. Have you checked that these cases aren't already covered?

**Review by tswhison — APPROVED** [review]

**Pull request merged**